x-pack/plugins/security_solution/cypress/tasks/alerts.ts

```diff
@@ -65,7 +68,6 @@ export const closeAlerts = () => {
 
 export const expandFirstAlertActions = () => {
   cy.get(TIMELINE_CONTEXT_MENU_BTN).should('be.visible');
-  cy.get(TIMELINE_CONTEXT_MENU_BTN).find('svg').should('have.attr', 'data-is-loaded');
```

Comment:
The `should allow a user with crud privileges to attach alerts to cases` test in `attach_alert_to_case.spec.ts` invokes the `expandFirstAlertActions()` test helper.

The test is failing on CI with the following error:

```
AssertionError: Timed out retrying after 120000ms: expected '<svg.euiIcon.euiButtonIcon__icon.css-e8xmlg-euiIcon-m-inherit>' to have attribute 'data-is-loaded'
    at expandFirstAlertActions (http://localhost:5620/__cypress/tests?p=cypress/integration/cases/attach_alert_to_case.spec.ts:22991:60)
    at Context.eval (http://localhost:5620/__cypress/tests?p=cypress/integration/cases/attach_alert_to_case.spec.ts:21750:50)
```

The `expandFirstAlertActions()` test helper in `cypress/tasks/alerts.ts` was updated ~ 21 hours ago with this change: <https://github.com/elastic/kibana/pull/135373/files#diff-9881e669a4305a15a96f3880d5d68906edfc15991e384a71d7ef5498ed04c0c8R68hh>

Based on desk testing, the `data-is-loaded` attribute:
- Only indicates whether or not the button's SVG is loaded (it's an implementaiton detail of EUI: <https://github.com/elastic/eui/blob/b151438410eb39a39c4540e04810a5023530d77f/src/components/icon/icon.tsx>)
- Does not seem to be reliable: When inspecting the DOM via dev tools, the attribute is not present, but the icon is clearly visible / loaded

Based on the git history, it looks like the `expandFirstAlertActions()` test helper didn't include this check:

```ts
cy.get(TIMELINE_CONTEXT_MENU_BTN).find('svg').should('have.attr', 'data-is-loaded');
```

in it's original implementation. The check was first added in this PR: <https://github.com/elastic/kibana/commit/683463ea43db9fc7ba2be43c56d2d059018e90c5#diff-9881e669a4305a15a96f3880d5d68906edfc15991e384a71d7ef5498ed04c0c8R68>

Since this check only determines whether or not the SVG was loaded in the button icon, and `data-is-loaded` does not appear to be reilable, I'm removing this `cy.get`.

Reply:
Sounds good 👍 

I was the one that actually added that check, so for context, this is the comment that details the change: https://github.com/elastic/kibana/pull/130072#discussion_r855616111

When I ran into this I had no issue with it passing locally, but it would fail on CI, so you may have to click-loop and/or wait for the menu if there's still an issue waiting on the table to load before the icon is visible to click.